src/Service.GraphQLBuilder/GraphQLTypes/SupportedTypes.cs

```diff
@@ -8,7 +8,7 @@ namespace Azure.DataApiBuilder.Service.GraphQLBuilder.GraphQLTypes
     /// The type names mentioned here are Hotchocolate scalar built in types.
     /// The corresponding SQL type name may be different for e.g. UUID maps to Guid as the SQL type.
     /// </summary>
-    public static class SupportedTypes
+    public static class SupportedHotChocolateTypes
```

Comment:
I am confused on what this class represents. Are these all of the support hot chocolate types and also only the types that are exclusive to SQL? But then in addition to that, those types listed as GraphQL Types, while supported in SQL are potentially mapped to a different SQL type than is the type name we see in Hot Chocolate?

I find this layout a bit confusing and not sure how easy it will be to extend and re-use given that confusion.

Reply:
Renamed the class/added comment.